src/test/java/com/easyrest/tests/BaseTest.java

```diff
@@ -43,10 +52,12 @@ public void setUp() {
         extent.attachReporter(spark);
     }
 
-    public void getScreenShotPath(String testCaseName) throws IOException {
+   public void getScreenShotPath(String testCaseName, WebDriver driver) throws IOException {
```

Comment:
why do you need to pass a driver as a parameter? you already have an instance as field of BaseTest class

Reply:
> why do you need to pass a driver as a parameter? you already have an instance as field of BaseTest class

agree, do not need this, fix